styles.css

```diff
@@ -0,0 +1,46 @@
+body {
+  background-color: lightblue;
+  position: relative;
+  text-align: center;
+  font-family: sans-serif;
```

Comment:
please write the font family before writing sans-serif for example; font-family: 'Arial', 'sans-serif'